app/experimenter/legacy-ui/rapid/components/visualization/Overview.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import {
+  displaySelectOptionList,
+  displaySelectOptionLabels,
+} from "experimenter-rapid/components/experiments/utils";
+import {
+  featureOptions,
+  audienceOptions,
+  firefoxVersionOptions,
+  firefoxChannelOptions,
+} from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import { ExperimentData } from "experimenter-types/experiment";
+
+const Overview: React.FC<{
+  experimentData: ExperimentData;
+}> = ({ experimentData }) => {
+  const targetingValues = [
+    {
+      data: experimentData.firefox_min_version,
+      options: firefoxVersionOptions,
+    },
+    { data: experimentData.firefox_channel, options: firefoxChannelOptions },
+    { data: experimentData.audience, options: audienceOptions },
+  ];
+  return (
+    <div className="mb-5">
+      <table className="table text-left h5 m-0">
+        <tbody>
+          <tr>
+            <th scope="col">
+              <p className="text-uppercase">Targeting</p>
+              {targetingValues.map((target) => {
+                return (
+                  <p key={target.data} className="h6">
+                    {displaySelectOptionLabels(target.options, target.data)}
+                    {target.data === experimentData.firefox_min_version
+                      ? "+"
+                      : ""}
+                  </p>
+                );
+              })}
+            </th>
+            <th scope="col">
+              <p className="text-uppercase">Probe Sets</p>
+              {displaySelectOptionList(
+                featureOptions,
+                experimentData.features,
+              ).map((feature) => {
+                return (
+                  <p key={feature} className="h6">
+                    {feature}
+                  </p>
+                );
+              })}
```

Comment:
Since you're not performing any JS operations before render you could just return the JSX shorthand style (same with `targetingValues`):

```tsx
).map((feature) => (
  <p key={feature} className="h6">
    {feature}
  </p>
))} 
```

Not a blocker though; using explicit return is totally valid.